operations/system_management_health/Remove_Kiali.md

```diff
@@ -0,0 +1,23 @@
+# Remove Kiali
+
+If not planning to use Kiali, then Kiali may be removed for CVE (Common Vulnerabilities and Exposures) remediation.
+**NOTE:** The removal will not persist after a CSM upgrade, so the removal procedure must be rerun after CSM upgrades.
+
+## Procedure
+
+1. Delete `kiali` deployment and `cray-kiali` chart.
+
+    ```bash
+    ncn-m# kubectl delete deployment kiali -n istio-system
```

Comment:
Remove the prompts to follow the new main branch conventions: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts-inside-of-directions